src/game.cpp

```diff
@@ -228,7 +220,7 @@ void Game::drawBoard() {
   std::cout << "  └";
 
   for (int i = 0; i < gameBoardPlaySize; i++) {
-    std::cout << border;
+    std::cout << horizontalBorder;
```

Comment:
Everytime you want to draw the `[horizontalBorder string]`, you would just call the function.